scripts/qiita

```diff
@@ -306,7 +308,14 @@ def start(port, master):
         else:
             raise
     base_url = qiita_config.base_url
-    click.echo("Qiita started on %s:%d @ %s" % (base_url, port, ctime()))
+
+    if re.search(':\d+$', base_url):
+        tmp_str = "Qiita started on %s @ %s" % (base_url, ctime())
```

Comment:
We normally try to avoid using the data type in the variable name, for example we try to _not_:
```
my_list = ['1']
var_str = 'dfasdf'
a_set = set([1, 1 ,1, 2])
```